parameters/query/mockup/company.yaml

```diff
@@ -0,0 +1,13 @@
+- in: query
+  name: count
+  description: |
+    Set it to a value greater than `1` to receive multiple data.
```

Comment:
Instead of _multiple data_ use something like _sequence-like_

Reply:
whats wrong with "multiple data"

Reply:
The wording sounds clunky imo. I could be nitpicking tho :D 
What I am getting at is, that it should be clear that the user will receive an iterable of the return type.
At least that's how I interpret it and that's probably how it will be used in the language of choice.

In python for example it would be a type `typing.Sequence[int]`

Reply:
you will get an array of objects instead of a single object, that should be clear with the wording "multiple data". The `count` is a query parameter and has nothing to do with a specific language.

Reply:
Mhm that's what I meant. array of objects, that already sounds nicer.
But if `multiple data` is universally understood as "you get an array back" then I take everything back.
I just thought the wording feels a bit odd.